os/fs/smartfs/smartfs_smart.c

```diff
@@ -1938,75 +1933,21 @@ int smartfs_rename(struct inode *mountpt, const char *oldrelpath, const char *ne
 	oldentry.name = NULL;
 	newentry.name = NULL;
 	ret = smartfs_finddirentry(fs, &oldentry, oldrelpath);
-	if (ret < 0) {
+	if (ret != OK) {
```

Comment:
can we check if (ret)

Reply:
writing (ret != OK) makes it understandable as checking if the function execution is not OK, then exit the operation.

Done in the same way for all smartfs fs operations.

Reply:
It might reduce instructions used

Reply:
@Taejun-Kwon 
This might be a good point? Extra instructions might be required to resolve the value of OK macro ?

However, since this is done in the same way at all instances in smartfs, changes will be done altogether only if required in future commits.

Reply:
@sangam-swami  Your suggestion will work fine here, but smartfs uses mtd operation many times, these mtd operation returns written bytes, or read bytes or number of pages in sector...
In this case it will return positive value, so I requested to him to use OK explicitly to get correct result.